phpunit.xml

```diff
@@ -6,7 +6,9 @@
     <env name="DTT_API_URL" value="http://localhost"/>
     <env name="DTT_MINK_DRIVER_ARGS" value='["chrome", null, "http://localhost:4444/wd/hub"]'/>
     <env name="DTT_API_OPTIONS" value='{"socketTimeout": 360, "domWaitTimeout": 3600000}' />
-    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="web/sites/default/files/browser_output"/>
+    <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/app/web/sites/default/files/browser_output"/>
```

Comment:
We are using this file to run on CircleCI. 
`/app/web/sites/default/files/browser_output` will be not work.

Reply:
@tannguyen04 ah yep! Were you using browser output before in CircleCI, I didn't think Circle was using it.

Reply:
we are not using output on CircleCi. But would like change it. I think We no need use absolute path for it

Reply:
@tannguyen04 I'll do a follow up PR after this one. To work in CircleCI it will need a way to look at the result files.